### ISSUE TITLE
table-read/sequence does not accept sequence of records

### ISSUE THREAD

**Issue by samdphillips** [massung/tabular-asa#1]
According to the [documentation](https://docs.racket-lang.org/tabular-asa/index.html#%28def._%28%28lib._tabular-asa%2Fmain..rkt%29._table-read%2Fsequence%29%29) `table-read/sequence` should accept a sequence of records.

```scheme
> (table-read/sequence (list (hasheq 'a 1)))
../../../Library/Racket/snapshot/pkgs/tabular-asa/read.rkt:83:4: result arity mismatch;
 expected number of values not received
  expected: 1
  received: 2
```

**Comment by massung** [comment]
I know this is old @samdphillips. I had GitHub notifications muted for a long time. I'm making updates to tabular-asa now and will look at this and your `table-groupby` bug as well as see what I find.

**Comment by massung** [comment]
I have a fix for this. Adding tests.

**Comment by massung** [comment]
Fixed by f0c8790.
